client/src/pages/events.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import '../style/events-page.css';
+import '../style/dropdown.css';
+import '../style/m-button.css';
+import { CardDeck, Container } from 'react-bootstrap';
+import EventCard from '../components/event-card';
+import 'react-datepicker/dist/react-datepicker.css';
+import LocalSearch from '../components/local-search';
+import DropdownGroup from '../components/dropdown-group';
+import Api from '../services/api';
+
+class Events extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      events: [],
+    };
+  }
+
+  componentDidMount() {
+    this.getAll();
+  }
+
+  getAll() {
+    Api.getAll('events')
+      .then((response) => {
+        if (response.error) {
+          // eslint-disable-next-line no-console
+          console.error(response);
+          return;
+        }
+        this.setState({
+          events: response.data,
+        });
+      });
+  }
+
+  render() {
+    const { events } = this.state;
+
+    return (
+      <Container fluid className="page-container p-0">
```

Comment:
![image](https://user-images.githubusercontent.com/6152066/76622330-fff4dc80-6539-11ea-89ca-7978097f3797.png)

What will happen when database is empty or server return error, or network brakes down? I think, you don't need to change this PR, but you can show some `Loading message` or spinner in this case
https://react-bootstrap.github.io/components/spinners/